component/file_cache/lru_policy.go

```diff
@@ -387,9 +426,28 @@ func (p *lruPolicy) deleteExpiredNodes() {
 	}
 
 	for ; node != nil && count < p.maxEviction; node = node.next {
-		delItems = append(delItems, node)
-		node.deleted = true
-		count++
+		p.fileOps.Range(func(key, value interface{}) bool {
```

Comment:
This Range function seems to be replicated a few times. Do you think it would be worth having a separate function for that?